arraycontainer.go

```diff
@@ -439,6 +439,7 @@ func (ac *arrayContainer) lazyorArray(value2 *arrayContainer) container {
 }
 
 func (ac *arrayContainer) and(a container) container {
+	//p("ac.and() called")
```

Comment:
I do like the kernel of this idea; making it easier to shift back and forth.

In practice, I see issues like:

1. we don't have a clean bijection. Inverting the transformation is difficult, because you don't know which fmt.Printf statements should now be commented, and which should not because they were never comments to begin with. One could try to surmount this by decorating the tail of the statement with another comment, but how to pick a comment that won't also have false positives?
2. One frequently wants to uncomment a single/small number of prints, and leave the rest alone. The conversion tool would be a blunt all or nothing per file hammer. I suppose this could be handled by an editor macro, but... starting to sound like lots of work to write a tool and editor macros. Compared to the simplicity of //p this seems crazy.
3. People would have to be educated about the conversion tool as well; this is big
barrier to entry. Any tooling has a high bar to meet compared to the simplicity of "position the cursor on the comment and hit the delete key twice", which is pretty darn easy way to activate a logging statement.